general-concepts/mirrors/text.xml

```diff
@@ -100,23 +100,24 @@ devspace.
 </p>
 
 <p>
-Previous policy was to use <c>mirror://gentoo</c> directly, but this is now deprecated, as that
-wouldn't allow to have long-term availability and traceability of the source files, which might be a
-requirement of the license.
+Previous policy was to use <c>mirror://gentoo</c> directly, but this is now
+deprecated, as that wouldn't allow to have long-term availability and
+traceability of the source files, which might be a requirement of the license.
 </p>
 
 <p>
-When you upload the file to <c>dev.gentoo.org:~/public_html</c>, you must ensure that it, and its
-parent directories, are world-readable. An example <c>SRC_URI</c> referencing
-a distfile mirrored this way is as follows:
+When you upload the file to <c>dev.gentoo.org:~/public_html</c>, you must ensure
```

Comment:
I thought we avoided "you".

Reply:
[Third person form should be used rather than first.](https://devmanual.gentoo.org/appendices/contributing/index.html#style-guidelines)

Nothing about second person. :smile: